File: KeychainIDFA-获取idfa标示当设备唯一识别,并保存到keychain中.基本不变.除非刷机./KeychainIDFA/ViewController.m
//
//  ViewController.m
//  KeychainIDFA
//
//  Created by Qixin on 14/12/18.
//  Copyright (c) 2014年 Qixin. All rights reserved.
//

#import "ViewController.h"
#import "KeychainIDFA.h"

//第一次:2d54d261-7bae-4014-8b81-3f9ff969b6e1
//第二次:2d54d261-7bae-4014-8b81-3f9ff969b6e1
//卸载app启动:2d54d261-7bae-4014-8b81-3f9ff969b6e1

//delete之后:2b8d8afc-7f87-4c9c-ac73-ec64a89fc1a8
//delete之后2次:2b8d8afc-7f87-4c9c-ac73-ec64a89fc1a8
//delete之后卸载:2b8d8afc-7f87-4c9c-ac73-ec64a89fc1a8

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
    //[KeychainIDFA deleteIDFA];
    NSLog(@"%@",[KeychainIDFA IDFA]);
}

@end

File: 新的插件商店 Saily/UnitTest/UnitTest/ViewController.swift
//
//  ViewController.swift
//  UnitTest
//
//  Created by Lakr Aream on 2019/7/26.
//  Copyright © 2019 Lakr Aream. All rights reserved.
//

import UIKit

let sharedObject = Listener()

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        sharedObject.regLinstenersOnMsgPass()
        
    }


}
